Assets/MixedRealityToolkit.Examples/Demos/Boundary/Scripts/BoundaryVisualizationDemo.cs

```diff
@@ -1,14 +1,11 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Core.Definitions.BoundarySystem;
-using Microsoft.MixedReality.Toolkit.Core.EventDatum.Boundary;
-using Microsoft.MixedReality.Toolkit.Core.Interfaces.BoundarySystem;
-using Microsoft.MixedReality.Toolkit.Core.Services;
+using Microsoft.MixedReality.Toolkit.Boundary;
 using System.Collections.Generic;
-using Microsoft.MixedReality.Toolkit.Core.Utilities.Async;
+using Microsoft.MixedReality.Toolkit.Utilities;
 using UnityEngine;
-using UnityEngine.Experimental.XR;
+using UBoundary = UnityEngine.Experimental.XR.Boundary;
```

Comment:
>UBoundary [](start = 6, length = 9)

nit: I think my preference would be to use Unity*, but i'll defer to the group/community's stance on this sort of naming.

Reply:
I think using a different namespace than Unity I best to help with ambiguity. 

Partly why `Inspector` was used instead of `Editor` etc.

Reply:
Sorry, do you mean different namespace than Editor? Or did you mean Chris' Unity* Prefix to Boundary to make UnityBoundary instead of UBoundary?

---
In reply to: [267121608](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267121608) [](ancestors = 267121608)

Reply:
I just used `Editor` as an example, but essentially a unique name so no prefixing is required.

I suppose it doesn't matter in the long run. `Unity*` is more direct and makes more sense imo